test/unit/mem/view/src/ViewSubViewTest.cpp

```diff
@@ -166,3 +167,69 @@ TEMPLATE_LIST_TEST_CASE("viewSubViewOffsetConstTest", "[memView]", alpaka::test:
 {
     alpaka::test::testViewSubViewOffsetConst<TestType, float>();
 }
+
+TEMPLATE_LIST_TEST_CASE("viewSubViewExample", "[memView]", alpaka::test::TestAccs)
+{
+    using Acc = TestType;
+    using Dev = alpaka::Dev<Acc>;
+    using Dim = alpaka::DimInt<2>;
+    using Idx = alpaka::Idx<Acc>;
+    using Vec = alpaka::Vec<Dim, Idx>;
+
+    auto const platform = alpaka::Platform<Acc>{};
+    auto const dev = alpaka::getDevByIdx(platform, 0);
+
+    auto const extent = Vec{4, 4};
```

Comment:
just a suggestion, could you make this non-square, to better show the rows and columns ?

Reply:
Ver good idea, done!